src/components/HeroSection.tsx

```diff
@@ -0,0 +1,46 @@
+import cn from '@lib/utils';
+import Image from 'next/image';
+import { H2 } from './ui/typography';
+
+type HeroSectionProps = {
+  className: string;
+};
+
+const HeroSection = ({ className }: HeroSectionProps) => {
+  return (
+    <section
+      className={cn(
+        'relative flex max-w-[81rem] flex-grow flex-col gap-10 px-3 pb-[17.5rem] pt-20 xl:items-center xl:p-0 xl:py-[7.625rem]',
+        className
+      )}
+    >
+      <Image
+        src="https://images.unsplash.com/photo-1622495893726-f1478dcebc03?q=100&w=1296&h=fit&fm=webp"
+        fill
+        alt="Hero Image"
+        className="left-0 top-0 -z-10 h-full w-full object-cover object-[10%_50%] xl:object-center"
+      />
```

Comment:
照片上面有一層濾鏡效果
1. `Image` 改成 `-z-20`
2. 新增一個 div 做效果
```suggestion
      <Image
        src="https://images.unsplash.com/photo-1622495893726-f1478dcebc03?q=100&w=1296&h=fit&fm=webp"
        fill
        alt="Hero Image"
        className="left-0 top-0 -z-20 h-full w-full object-cover object-[10%_50%] xl:object-center"
      />
      <div className="absolute inset-0 -z-10 bg-black bg-opacity-30" />
```

Reply:
fix on this commit: 75fb3d68c9678ba99d362222c9e74bb5b333bd61